activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Allow associations supporting the `dependent:` key to take `dependent: :destroy_later`.
+
+    ```ruby
+    class Account < ActiveRecord::Base
+        belongs_to :supplier, dependent: :destroy_later
```

Comment:
Let's rename the option to `destroy_async`, and the config and the job should have `async` instead of `later`.